data/formats-data.js

```diff
@@ -825,8 +825,8 @@ exports.BattleFormatsData = {
 		tier: "LC"
 	},
 	steelix: {
-		randomBattleMoves: ["stealthrock","earthquake","ironhead","roar","toxic","rockslide"],
-		randomDoubleBattleMoves: ["stealthrock","earthquake","ironhead","rockslide","protect","explosion","icefang","firefang"],
+		randomBattleMoves: ["stealthrock","earthquake","gyroball","roar","toxic","rockslide"],
+		randomDoubleBattleMoves: ["stealthrock","earthquake","gyroball","rockslide","protect","explosion","icefang","firefang"],
```

Comment:
Iron Head is preferred due to Sheer Force and because there are no max speed Pokemon so Gyro Ball is weaker.